CHANGELOG.rst

```diff
@@ -1,50 +1,3 @@
-0.7.5 (2015-01-21)
-------------------
-- Handle request path parameters with spaces correctly
-- Performance improvements for loading large api docs
-- Misc bug fixes
-
-0.7.4 (2014-12-11)
-------------------
-- Requests urlencode params as utf8
-- Docs related to 0.7.2
-- Declare utf-8 encoding for all files
-
-0.7.3 (2014-12-11)
-------------------
-- request logging is now done on the debug level instead of
-  info level.
-
-0.7.2 (2014-12-11)
-------------------
-- Allow headers to be passed in the api_docs request
-
-0.7.1 (2014-12-11)
-------------------
-- Requests no longer mutate clients
-
-0.7.0 (2014-11-26)
-------------------
-- headers are no longer cached and required as part of async and
-  http client setup.
-
-0.6.0 (2014-10-30)
-------------------
-- format='date' params are now represented and passed as
-  datetime.date objects instead of datetime.datetimes.
-
-0.5.7 (2014-10-23)
-------------------
-- Successfully validate objects that have additional fields beyond those
-  specified in the associated model.
-- Store raw deserialized JSON in '_raw' attribute of objects.
-- Ignore '_raw' attribute in object equality checks.
-
-0.2.0 (2013-10-28)
-------------------
-- Add close() methods to client and http_client.
-
-0.1.0 (2013-10-24)
-------------------
-
-- Initial release
+0.1.0-rc1 (2015-05-13)
```

Comment:
Just one question about versions.

The git repo currently has tags for all of our swagger-py release, so if we start again at 0.1, we're going to overlap.

It would probably be good to keep the tags instead of erasing them entirely, because we do still have people using the old versions for now at least.

Should we maybe call this 1.0.0-rc1 ?